### PR TITLE
MDEXP-247

### PR DESCRIPTION
**Pull request by Igor-Gorchakov**
Data Migration issue on goldenrod
https://issues.folio.org/browse/MDEXP-247

## Purpose
The purpose is to fix and verify the error happens while running data migration script to update from v1.1.1 to the latest master v2.2.0-SNAPSHOT
Verification details are in Jira ticket

## Approach
The approach is to move the logic for creating default profiles into schema.json

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/main/resources/templates/db_scripts/migration_scripts/create_default_job_profile.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
+INSERT INTO
```
Comment (KVupp) [inline src/main/resources/templates/db_scripts/migration_scripts/create_default_job_profile.sql:1]: These are not migration scripts , so let's move it to a common folder

File: src/main/resources/templates/db_scripts/migration_scripts/create_default_mapping_profile.sql [hunk 2]
```diff
@@ -0,0 +1,20 @@
+INSERT INTO
+  ${myuniversity}_${mymodule}.mapping_profiles (id, jsonb)
+VALUES
```
Comment (KVupp) [inline src/main/resources/templates/db_scripts/migration_scripts/create_default_mapping_profile.sql:3]: same here, move it to a different folder

**Comment by KVupp** [comment]
Please add description to this ticket. and also the test results, that this has been tested from v1.1.1 to current version.(While testing make sure that there is data in job_executions table)

File: src/main/resources/templates/db_scripts/migration_scripts/create_default_job_profile.sql (hunk 1 above)
Comment (Igor-Gorchakov) [reply src/main/resources/templates/db_scripts/migration_scripts/create_default_job_profile.sql]: moved to the "default_data" folder

File: src/main/resources/templates/db_scripts/schema.json [hunk 3]
```diff
@@ -67,6 +67,16 @@
       "snippetPath": "migration_scripts/rename_progress_current_to_exported.sql",
       "fromModuleVersion": "mod-data-export-2.0.0"
     },
+    {
+      "run": "after",
+      "snippetPath": "default_data/create_default_mapping_profile.sql",
```
Comment (KVupp) [inline src/main/resources/templates/db_scripts/schema.json:72]: do we know if these scripts are executed in sequence? Please verify that

Comment (andrii-novytskyi) [reply src/main/resources/templates/db_scripts/schema.json:72]: ![image](https://user-images.githubusercontent.com/60778653/88692389-ad8e1400-d106-11ea-998a-41d8148a0332.png
Execution process from top to bottom

**Comment by Igor-Gorchakov** [comment]
> Please add description to this ticket. and also the test results, that this has been tested from v1.1.1 to current version.(While testing make sure that there is data in job_executions table)

Added details to Jira

**Pull request merged**